backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,61 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-no-phi-views
+      author: josh@skylight.digital
+      comment: Add views for some tables that omit columns containing PHI.
+      changes:
+        - createView:
+            viewName: api_user_no_phi_view
+            remarks: A subset of the api_user table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, updated_at, last_seen, is_deleted FROM ${database.defaultSchemaName}.api_user
+        - createView:
+            viewName: provider_no_phi_view
+            remarks: A subset of the provider table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, created_by, updated_at, updated_by, is_deleted, provider_id FROM ${database.defaultSchemaName}.provider
+        - createView:
+            viewName: person_no_phi_view
+            remarks: A subset of the person table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, created_by, updated_at, updated_by, is_deleted, organization_id, race, gender, ethnicity, lookup_id, county, state, employed_in_healthcare, role, resident_congregate_setting, facility_id, tribal_affiliation FROM ${database.defaultSchemaName}.person
```

Comment:
lookup_id should be excluded. At the time the requirements were written it was not being used, so it's null for most users, but we expect student ID numbers to go in there shortly, which are most def SPII.

Also, in nittiest of nitpicks, I would move `facility_id` up to the front next to `organization_id`—it's just at the end because postgresql doesn't respect the column-order directive in liquibase when you add new columns, but it's actually objectively more useful to have it up at the top in a `select *` situation, which I suspect metabase does a bunch of.

Reply:
Fixed in commit